erizo/src/erizo/SdpInfo.h

```diff
@@ -254,6 +258,11 @@ class SdpInfo {
   */
   const std::vector<Rid>& rids() const { return rids_; }
 
+  int getSessionVersion() const;
```

Comment:
I think it's always good to add support to SDP fields, but I'd do that in separate PRs to reduce the scope of fixes.

Reply:
ok I'll move out